static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -76,45 +84,83 @@
 
     @include mq($until: mobileLandscape) {
         font-size: 5vw;
-        padding-left: ($gs-gutter / 2);
-        padding-right: ($gs-gutter / 2);
+        line-height: 1.6;
         // Position after pseudo element which forces line break
         order: 1;
         justify-content: space-between;
     }
+
+    @include mq(mobileLandscape) {
+        font-size: 24px;
+    }
 }
 
+// Specific values for menu button breakpoints
+$veggieburgerSmall: 42px;
+$veggieburgerMedium: 52px;
+$gutterSmall: 30px;
+$gutterMedium: 37px;
+
 .new-header__nav__menu-button {
     // Override button from user agent stylesheet
-    background-color: $news-main-2;
+    position: relative;
     // Unset button from user agent stylesheet
+    cursor: pointer;
```

Comment:
Good spot I think maybe that's a remnant of Natalia building it with a different element previously.